doc/dev/dev_setup.md

```diff
@@ -7,7 +7,7 @@ or execute the various commands available in the toolbox.
 
 1.  If you don't already have it, install Python:
 
-    - Windows: https://www.python.org/downloads/ or from the windows store https://www.microsoft.com/p/python-37/9nj46sx7x90p
+    - Windows: https://www.python.org/downloads/ or from the [Windows store](ht tps://www.microsoft.com/p/python-37/9nj46sx7x90p)
```

Comment:
This link is broken. It's also pointing to python 3.7 - can we point to 3.9 or maybe just a general python page?